rep-0003.rst

```diff
@@ -180,6 +179,48 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2021)
+----------------------------------
+Required Support for:
+
+- Ubuntu Wily (15.10)
+- Ubuntu X (16.04)
+
+Minimum Requirements:
+
+- C++03
+
+  - Some C++11 will be enabled (TODO(esteve): need to determine which ones)
```

Comment:
Nothing concrete, but I guess that for example some APIs that accept a `boost::shared_ptr` would also take a `std::shared_ptr`. Though we could follow an incremental approach and see in each case what can be augmented to add C++11-specific features and if it's feasible.